CONTRIBUTING.md

```diff
@@ -1,5 +1,11 @@
 # Contributing to Node.js
 
+## Code of Conduct
+
+The Code of Conduct explains the *bare minimum* behavior
```

Comment:
Is bare minimum (with emphasis) required here? The language seems different than below.